### ISSUE TITLE
TypeError: _translate() takes 1 positional argument but 5 were given

### ISSUE THREAD

**Issue by wbszhu** [vanheeringen-lab/gimmemotifs#254]
Hi,
I use the command like below:
```
gimme motifs OUT-00-MS_PeQTLmotif.bed  OUT-00-MS-PeQTL_gimmotif/ -b OUT-00-LW_PeQTLmotif.bed -g  ~/22.genome/susScr11_xy/Sus_scrofa.Sscrofa11.1.dna.toplevel.fa --known --rawscore  -s 20
```
and the input format like this

head OUT-00-MS_PeQTLmotif.bed 
```
chr1	186731	186732	eqtl20_1	.
chr1	857255	857256	eqtl20_2	.
chr1	1522259	1522260	eqtl20_3	.
chr1	7612510	7612511	eqtl20_4	.
chr1	8934727	8934728	eqtl20_5	.
chr1	8934192	8934193	eqtl20_6	.
chr1	8934727	8934728	eqtl20_7	.
chr1	8934192	8934193	eqtl20_8	.
chr1	13447075	13447076	eqtl20_9	.
chr1	16248970	16248971	eqtl20_10	.
```
head OUT-00-LW_PeQTLmotif.bed 
```
chr1	186731	186732	eqtl20_1	.
chr1	857255	857256	eqtl20_2	.
chr1	1522259	1522260	eqtl20_3	.
chr1	7612510	7612511	eqtl20_4	.
chr1	8934727	8934728	eqtl20_5	.
chr1	8934192	8934193	eqtl20_6	.
chr1	8934727	8934728	eqtl20_7	.
chr1	8934192	8934193	eqtl20_8	.
chr1	13447075	13447076	eqtl20_9	.
chr1	16248970	16248971	eqtl20_10	.
```

and I received the error:
```
/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/pandas/core/arraylike.py:397: RuntimeWarning: divide by zero encountered in log10
  result = getattr(ufunc, method)(*inputs, **kwargs)
/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/pandas/io/formats/style.py:3558: RuntimeWarning: invalid value encountered in double_scalars
  norm = mpl.colors.Normalize(smin - (rng * low), smax + (rng * high))
/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/pandas/io/formats/style.py:3558: RuntimeWarning: invalid value encountered in double_scalars
  norm = mpl.colors.Normalize(smin - (rng * low), smax + (rng * high))
Traceback (most recent call last):
  File "/public/home/luzhang/miniconda3/envs/gimme/bin/gimme", line 11, in <module>
    cli(sys.argv[1:])
  File "/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/gimmemotifs/cli.py", line 746, in cli
    args.func(args)
  File "/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/gimmemotifs/commands/motifs.py", line 278, in motifs
    roc_html_report(
  File "/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/gimmemotifs/report.py", line 1018, in roc_html_report
    ExtraStyler(df)
  File "/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/pandas/io/formats/style.py", line 344, in render
    return self._render_html(sparse_index, sparse_columns, **kwargs)
  File "/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/pandas/io/formats/style_render.py", line 164, in _render_html
    d = self._translate(sparse_index, sparse_columns, max_rows, max_cols)
TypeError: _translate() takes 1 positional argument but 5 were given
```
Thanks
LuZhang

**Comment by siebrenf** [comment]
This looks like an error in pandas! Based on the error I think you are using pandas 1.4.x, but could you check which one exactly? 

If you aren't running the latest version of pandas (1.4.2) you could try that one, else you could try to install the previous version (1.3.5), and run the command with that.

**Comment by wbszhu** [comment]
Hi，
I am using the latest version of pandas (1.4.2)
conda list|grep pandas
···
pandas                    1.4.2            py39h1832856_1    conda-forge
···

**Comment by siebrenf** [comment]
Thanks! Hopefully they'll fix this in the next version!

For now, you can try using pandas 1.3.5

**Comment by wbszhu** [comment]
I just tried using pandas 1.3.5, but it still reports the same error as above.

**Comment by wbszhu** [comment]
Hi,
I tried pandas 1.1.5, Maybe it is ok.
conda list|grep "pandas"
```
pandas                    1.1.5            py39hde0f152_0    conda-forge

```
and the output dir
```
-rw-r--r-- 1 luzhang SHZhao  12K Apr 30 16:05 gimme.motifs.html
-rw-r--r-- 1 luzhang SHZhao  27K Apr 30 16:05 gimme.motifs.redundant.html
-rw-r--r-- 1 luzhang SHZhao 152K Apr 30 16:04 gimme.roc.report.txt
-rw-r--r-- 1 luzhang SHZhao  49K Apr 30 16:02 input.w20.bed
drwxr-xr-x 2 luzhang SHZhao 4.0K Apr 30 16:05 logos
drwxr-xr-x 2 luzhang SHZhao 4.0K Apr 30 16:05 motif_scan_results

```
and the log file(only some warning)
```
2022-04-30 15:59:44,058 - INFO - calculating stats
2022-04-30 16:00:56,698 - INFO - selecting non-redundant motifs
2022-04-30 16:00:58,610 - INFO - selected 1 non-redundant motifs: ROC AUC 1.000, PR AUC 1.000
2022-04-30 16:01:10,896 - INFO - determining FPR-based threshold
^Mscanning:   0%|          | 0/1398 [00:00<?, ? sequences/s]^Mscanning:   0%|          | 1/1398 [00:00<16:38,  1.40 sequences/s]^Mscanning: 100%|██████████| 
1398/1398 [00:00<00:00, 1854.43 sequences/s]
2022-04-30 16:01:15,432 - INFO - creating statistics report
/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/pandas/core/series.py:726: RuntimeWarning: divide by zero encountered in log10
  result = getattr(ufunc, method)(*inputs, **kwargs)
/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/pandas/io/formats/style.py:1130: RuntimeWarning: invalid value encountered in double_s
calars
  norm = colors.Normalize(smin - (rng * low), smax + (rng * high))
/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/pandas/core/series.py:726: RuntimeWarning: divide by zero encountered in log10
  result = getattr(ufunc, method)(*inputs, **kwargs)
/public/home/luzhang/miniconda3/envs/gimme/lib/python3.9/site-packages/pandas/io/formats/style.py:1130: RuntimeWarning: invalid value encountered in double_s
calars
  norm = colors.Normalize(smin - (rng * low), smax + (rng * high))
2022-04-30 16:02:06,585 - INFO - gimme motifs final report: OUT-00-MS-PeQTL_gimmotif/gimme.motifs.html
```

**Comment by gfudenberg** [comment]
Hi, I'm getting the same TypeError in python 3.9 with pandas 1.4.2, with installation via conda as suggested in the readme. 
```
  File "/home1/fudenber/.conda/envs/gimme/lib/python3.9/site-packages/pandas/io/formats/style_render.py", line 164, in _render_html
    d = self._translate(sparse_index, sparse_columns, max_rows, max_cols)
TypeError: _translate() takes 1 positional argument but 5 were given
```

Is the suggested solution still downgrading pandas to 1.1.5, @siebrenf ?

Thanks!

**Comment by siebrenf** [comment]
This issue was fixed in gimmemotifs 0.17.2. 0.17.2 and 0.18.0 are now available on both conda and pypi!
